conans/client/command.py

```diff
@@ -1089,6 +1112,8 @@ def _commands(self):
             if not method_name.startswith('_'):
                 if "export_pkg" == method_name:
                     method_name = "export-pkg"
+                if "execute" == method_name:
```

Comment:
I would say, to keep the command and method named exactly the same. The export-pkg is about consistency of underscore and dash, but otherwise, better be the same.

Reply:
Sure, but we cannot use `exec` as command name then given that it is a reserved keyword in python.